arangod/Graph/TraverserOptions.h

```diff
@@ -204,7 +204,7 @@ struct TraverserOptions : public graph::BaseOptions {
 
   void initializeIndexConditions(
       aql::Ast* ast,
-      std::unordered_map<aql::VariableId, aql::VarInfo> const& varInfo,
```

Comment:
As a suggestion: It seems that variableId->VarInfo Map is quite common construct. Would it be possible to make this some global "using" for type? And avoid writing full type everywhere. As we are touching all this places in this PR anyway.

Reply:
Sounds good for me.
But it more changes in Graph (needs additional includes, now forward declaration), Markus was against it

Reply:
😞

Reply:
I will make it for all files except Graph directory

Reply:
This does not make sense. It would leave the codebase in an inconsistent state.

If you make this change, it needs to be a separate PR with a description saying what it does.

Reply:
Ok, I will do it for Graph too.

Reply:
Done